aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -44,14 +45,28 @@
 
 namespace sdp {
 namespace {
+
+// TODO(eqy): more benchmarking to determine whether this should include sm86/89
+// Needs to be kept in-sync with test_fused_chocie in test_transformers.py
+bool check_prefer_cudnn_attention() {
```

Comment:
I actually think we should also add a check on the cuDNN version and ensure that it is greater than 9

Reply:
I think that this check might actually be the problem. This macro is checking that the cudnn-frontend version is > 9. right?
I think what we want is a way at runtime to see if the backend version is > 9